<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>750</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>610</width>
    <height>750</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>610</width>
    <height>760</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
font:  11pt &quot;Segoe UI Variable Display Semib&quot;;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>15</number>
     </property>
     <property name="bottomMargin">
      <number>20</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="background_label">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>110</width>
           <height>100</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>../../Build2/img/imgmedium.jpg</pixmap>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="hello_label">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
font: 600 18pt &quot;Segoe UI Variable Display Semib&quot;;</string>
           </property>
           <property name="text">
            <string>Добро пожаловать </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="name_label">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
font: 600 15pt &quot;Segoe UI Variable Display Semib&quot;;</string>
           </property>
           <property name="text">
            <string>tina</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="background_label_2">
         <property name="maximumSize">
          <size>
           <width>110</width>
           <height>100</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>../../Build2/img/imgmedium.jpg</pixmap>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">color: transfert</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>30</number>
         </property>
         <item>
          <widget class="QPushButton" name="openAll_Button">
           <property name="minimumSize">
            <size>
             <width>380</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>330</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 0, 0);
border:2px solid rgb(85, 255, 255);
border-radius:18px;
color: rgb(185, 255, 253)
</string>
           </property>
           <property name="text">
            <string>показать все</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="addNew_Button">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>380</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 0, 0);
border:2px solid rgb(85, 255, 255);
border-radius:18px;
color: rgb(185, 255, 253)
</string>
           </property>
           <property name="text">
            <string>Создать новый</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="delete_Button">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>380</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 0, 0);
border:2px solid rgb(85, 255, 255);
border-radius:18px;
color: rgb(185, 255, 253)
</string>
           </property>
           <property name="text">
            <string>Удалить существующий</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="delete_Button_2">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>380</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 0, 0);
border:2px solid rgb(85, 255, 255);
border-radius:18px;
color: rgb(185, 255, 253)
</string>
           </property>
           <property name="text">
            <string>изменить существующий</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="log_Button">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>380</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 0, 0);
border:2px solid rgb(85, 255, 255);
border-radius:18px;
color: rgb(185, 255, 253)

</string>
           </property>
           <property name="text">
            <string>открыть журнал</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="maximumSize">
          <size>
           <width>110</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>100</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>../../Build2/img/img_red_mini.jpg</pixmap>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_4">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:rgb(0, 0, 0);
color:rgb(225, 225, 225);

border:2px solid rgb(255, 200, 89);
border-radius:18px;

</string>
         </property>
         <property name="text">
          <string>выйти</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>100</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>../../Build2/img/img_red_mini.jpg</pixmap>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
